.cirrus.yml

```diff
@@ -118,6 +116,17 @@ benchmark_task:
   only_if: ${SKIP_BENCHMARK_TASK} == ""
   auto_cancellation: true
   name: "${CIRRUS_OS}: performance benchmarking"
+  # Custom clone behaviour to enable ASV to access the PR base branch (if on a
+  #  PR).
+  clone_script: |
+    if [ -z "$CIRRUS_PR" ]; then
+      git clone --branch=$CIRRUS_BRANCH https://x-access-token:${CIRRUS_REPO_CLONE_TOKEN}@github.com/${CIRRUS_REPO_FULL_NAME}.git $CIRRUS_WORKING_DIR
+      git reset --hard $CIRRUS_CHANGE_IN_REPO
+    else
+      git clone --recursive https://x-access-token:${CIRRUS_REPO_CLONE_TOKEN}@github.com/${CIRRUS_REPO_FULL_NAME}.git $CIRRUS_WORKING_DIR
+      git fetch origin pull/$CIRRUS_PR/head:pull/$CIRRUS_PR
```

Comment:
Should `CIRRUS_PR` not be in curly brackets here?

Reply:
Since our use case allowed me to borrow wholesale from [Cirrus' own example](https://cirrus-ci.org/guide/tips-and-tricks/#custom-clone-command), I never thought about that. Certainly looks like it should be as you say. We can't really test whether it works until it's merged!

ff01df5b6da60850d8dcc8ee5f5813c8e314fa0b